text/2021-09-27-refine-error-handling.md

```diff
@@ -0,0 +1,293 @@
+# Refine Error Handling
+
+## Background
+
+There are a lot of different error handling pattern in Chaos Mesh. Some of the
+errors in the code are equipped with a backtrace, while some are not. The mass
+in error handling has been blocking us from diagnosing the problem and handling
+potential errors. The Chaos Mesh code base imports `github.com/pingcap/errors`,
+`github.com/pkg/errors` and the `errors` in standard library.
```

Comment:
Since the big difference between these libraries , I think an advice about how to adapt them is necessary.